recipes/treelite/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "treelite" %}
+{% set version = "0.32" %}
+{% set sha256 = "ac34d0f925dad06d1c90932a57e805e6f0361988f47a457936997fb3aad17cfa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
```

Comment:
I can see there is no source tarball on pypi so that's not an option, but you should use the release on github rather than the actual repo:

https://github.com/dmlc/treelite/archive/0.32.tar.gz

Reply:
Thank you for your review. At the moment, the tar.gz for release 0.32 is missing dmlc-core source (which is included with git clone --recursive).

Reply:
we already have ``dmlc`` on ``conda-forge``, can't you use it instead of getting from submodules?

Reply:
@marcelotrevisani indeed, seems like the best solution. It will require some rewrite of treelite CMakeLists.txt to include dmlc (and making sure that dmlc has exported CMake targets).
As I'm not the maintainer of those packages, I might leave the PR as is for the moment.